### PR TITLE
updated certain tasks to run on Machine Group as well as Agent (#3103)

### PR DESCRIPTION
**Pull request by arjgupta**
* updated certain tasks to run on Machine Group as well as Agent

* updated some more tasks to run on machine group

* removed ms build from the list of tasks to be run on machine group

* updated task patch version

* updated patch versions

* updated quickperftest/task.loc.json

### CODE REVIEW

File: Tasks/QuickPerfTest/task.loc.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
       "type": "connectedService:Generic",
       "label": "ms-resource:loc.input.label.connectedServiceName",
       "defaultValue": "",
-      "required": true,
+	  "required": true,
```
Comment (kmkumaran) [inline Tasks/QuickPerfTest/task.loc.json]: Can you avoid changes that are not part of this PR.

File: Tasks/VsTest/task.json [hunk 2]
```diff
@@ -9,6 +9,10 @@
         "Build",
         "Release"
     ],
+	"runsOn": [
```
Comment (kmkumaran) [inline Tasks/VsTest/task.json]: Patch version update?

Comment (arjgupta) [inline Tasks/VsTest/task.json]: Updated
